content/en/docs/tasks/security/tls-configuration/workload-min-tls-version/index.md

```diff
@@ -86,6 +86,19 @@ Cipher is (NONE)
 
 ## Cleanup
 
+Delete sample applications `sleep` and `httpbin` from `foo` namespace:
+
+{{< text bash >}}
+$ kubectl delete -f samples/httpbin/httpbin.yaml -n foo
+$ kubectl delete -f samples/sleep/sleep.yaml -n foo
+{{< /text >}}
+
+Uninstall Istio from cluster:
+
+{{< text bash >}}
+$ istioctl uninstall --revision=default
```

Comment:
It does look like the test runs this command, but you might change to the more common `istioctl uninstall --purge -y` in the instructions and the test.